improver/calibration/reliability_calibration.py

```diff
@@ -623,6 +634,48 @@ def _calculate_reliability_probabilities(self, reliability_table):
 
         return forecast_probability, observation_frequency
 
+    @staticmethod
+    def _interpolate(forecast_threshold, reliability_probabilities,
+                     observation_frequencies):
+        """
+        Perform interpolation of the forecast probabilities using the
+        calibration data to produce the calibrated forecast. Where necessary
```

Comment:
Should `calibration data` be something like `reliability table information`?

Reply:
Done.